pkgs/applications/editors/neovim/utils.nix

```
@@ -176,7 +176,9 @@ let
         (lib.replaceStrings [ "-" ] [ "_" ])
       ];
 
-      nvimGrammars = lib.mapAttrsToList (name: value: value.origGrammar) vimPlugins.nvim-treesitter.grammarPlugins;
+      nvimGrammars = lib.mapAttrsToList (name: value: value.origGrammar or builtins.throw
+          "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first"
+        ) vimPlugins.nvim-treesitter.grammarPlugins;
```

let's format it with nixfmt
```suggestion
      nvimGrammars = lib.mapAttrsToList (
        name: value:
        value.origGrammar or builtins.throw
          "additions to `pkgs.vimPlugins.nvim-treesitter.grammarPlugins` set should be passed through `pkgs.neovimUtils.grammarToPlugin` first"
      ) vimPlugins.nvim-treesitter.grammarPlugins;
```

---

I had the wrong formatter set up apparently and it screwed up the whole file lol. thanks for doing that for me :)